sagemaker-neo-notebooks/edge/cpp-integration/README.md

```diff
@@ -0,0 +1,219 @@
+
+# C++ Tutorial
+
+## Prerequisite
+
+Please review the following prerequisites tasks before proceed with tutorial
+
+### Environment Setup
+
+#### Environment Setup for Windows
+
+In order to compile DLR and tutorial dependencies, please install [Visual Studio 2019](https://visualstudio.microsoft.com/vs/).
+
+Additionally, please make sure msbuild path is setup properly for PowerShell. In order to do so, please append your msbuild path, such as C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin, to your system path.
+
+### Install CMake
```

Comment:
This can be moved under *Prerequisite Software list*

Reply:
it's revised.